_pages/views.md

```diff
@@ -1091,6 +1123,38 @@ The listGrid widget offers the following event handlers:
 * onEditedHandler - what events to perform after a row is edited
 * onSelectedHandler - what events to perform after a row is selected
 
+#### listGrid Example
+
+For a listGrid, there must be an association to the document you want to show 
+inside the document you want to show them from. 
+
+```xml
+
+<association name="contact" type="aggregation" required="true">
+	<displayName>Contact</displayName>
+	<description>The contact details for the user.</description>
+	<documentName>Contact</documentName>
+	<queryName>qContacts</queryName>
+</association>
+
+```
+
+Once your association has been added in, you can add in the listGrid
```

Comment:
Hey @brandon-klar, the association is not required for a `ListGrid`. It is only required for filtering.